.github_changelog_generator

```diff
@@ -1,4 +1,4 @@
-future-release=v0.27.0
-since-tag=v0.26.1
+future-release=v0.28.0
+since-tag=v0.27.0
```

Comment:
when did we release v0.27.0? I don't see it in the release page

Reply:
https://github.com/mongodb/go-client-mongodb-atlas/releases/tag/v0.27.0

Reply:
ah you released it now xD